examples/simple-generator/README.adoc

```diff
@@ -0,0 +1,26 @@
+# Example: Simple Yeoman Generator For Fiori Elements
+A simplified example illustrating how to create a yeoman generator using the `fiori-elements-writer` and the `axios-extension`.
+
+## Example
+This example project contains a generator that asks a few simple questions and then uses the `@sap-ux/fiori-elements-writer` to generate a Fiori application. Additionally it generates a karma config for the project utilzing our new mockserver middleware.
```

Comment:
```suggestion
This example project contains a generator that asks a few simple questions and then uses the `@sap-ux/fiori-elements-writer` to generate a Fiori application. Additionally it generates a karma config for the project utilizing our new mockserver middleware.
```
Also:
* `our new mockserver middleware` -> `the new mockserver middleware`?
* add a link to the new mockserver?